wgpu-types/src/lib.rs

```diff
@@ -862,7 +862,7 @@ bitflags::bitflags! {
         ///
         /// This mainly applies to a Vulkan driver's compliance version. If the major compliance version
         /// is `0`, then the driver is ignored. This flag allows that driver to be enabled for testing.
-        const ALLOW_NONCOMPLIANT_ADAPTER = 1 << 3;
+        const ALLOW_UNDERLYING_NONCOMPLIANT_ADAPTER = 1 << 3;
```

Comment:
Could we clarify the doc comment, on rereading the underlying-ness isn't clear.

Either here or in a followup.

Reply:
Pushed another commit, feel free to merge if it looks good.